plugins/modules/mas.py

```diff
@@ -115,6 +120,7 @@ def __init__(self, module):
         # Initialize data properties
         self.mas_path = self.module.get_bin_path('mas')
         self._checked_signin = False
+        self._mac_version = platform.mac_ver()[0]
```

Comment:
```suggestion
        self._mac_version = platform.mac_ver()[0] or '0.0'
```
This avoids strange errors (`AttributeError: 'LooseVersion' object has no attribute 'vstring'`) when `self._mac_version == ''`. This shouldn't happen in real life, but might happen during tests.